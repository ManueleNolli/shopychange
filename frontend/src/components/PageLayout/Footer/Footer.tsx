import React from 'react'
import { Box, Image, Stack, Text, Divider } from '@chakra-ui/react'
import { Logo } from '../../../assets/AssetsManager'

export default function Footer() {
  return (
    <Box mt="10">
      <Divider />
      <Stack
        minW={'100%'}
        py={4}
        direction={{ base: 'column', md: 'row' }}
        spacing={4}
        justify={{ base: 'center', md: 'center' }}
        align={{ base: 'center', md: 'center' }}
      >
        <Image src={Logo} alt="Logo" w={{ base: '8vw', md: '4vw' }} />
        <Text>Â© 2023 SUPSI by Manuele Nolli. All rights reserved</Text>
      </Stack>
    </Box>
  )
}
