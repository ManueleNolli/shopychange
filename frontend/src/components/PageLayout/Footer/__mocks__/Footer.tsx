import { divWithoutChildrenMock } from '../../../../../mocksUtils'

const FooterMock = () => divWithoutChildrenMock('Footer')
export default FooterMock
