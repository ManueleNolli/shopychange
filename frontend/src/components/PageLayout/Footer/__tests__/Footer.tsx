import React from 'react'
import { render } from '@testing-library/react'
import Footer from '../Footer'

describe('Footer', () => {
  test('renders correctly', () => {
    const tree = render(<Footer />)

    expect(tree.container).toMatchSnapshot()
  })
})
