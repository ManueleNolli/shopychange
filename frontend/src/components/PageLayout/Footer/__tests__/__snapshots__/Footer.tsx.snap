// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <div
    class="css-1q7njkh"
  >
    <hr
      aria-orientation="horizontal"
      class="chakra-divider css-1upb9tn"
    />
    <div
      class="chakra-stack css-10bjzzg"
    >
      <img
        alt="Logo"
        class="chakra-image css-waj431"
        src="logo.png"
      />
      <p
        class="chakra-text css-0"
      >
        Â© 2023 SUPSI by Manuele Nolli. All rights reserved
      </p>
    </div>
  </div>
</div>
`;
